demo/NostrSDKDemo/Demo Views/QueryRelayDemoView.swift

```diff
@@ -32,8 +42,12 @@ struct QueryRelayDemoView: View {
                 }
 
                 Picker("Kind", selection: $selectedKind) {
-                    ForEach(kindOptions, id: \.self) { number in
-                        Text("\(number)")
+                    ForEach(kindOptions.keys.sorted(), id: \.self) { number in
+                        if let name = kindOptions[number] {
+                            Text("\(name) (\(number))")
```

Comment:
It's a bit unusual to display kind numbers formatted (with separators) as it's formatted differently depending on the locale, and the NIPs repo doesn't do that.

Perhaps something like this would be better?

```swift
if let name = kindOptions[number] {
    Text("\(name) (\(String(number)))")
} else {
    Text(String(number))
}
```

Reply:
Agree, thanks.